meshsync/exec.go

```diff
@@ -43,18 +43,56 @@ func (h *Handler) processExecRequest(obj interface{}, cfg config.ListenerConfig)
 			if !bool(req.Stop) {
 				h.channelPool[id] = channels.NewStructChannel()
 				h.Log.Info("Starting session")
+
+				err := h.Broker.Publish("active_sessions.exec", &broker.Message{
+					ObjectType: broker.ActiveExecObject,
+					Object:     h.getActiveChannels(),
+				})
+				if err != nil {
+					h.Log.Error(ErrGetObject(err))
+				}
 				go h.streamSession(id, req, cfg)
 			}
 		} else {
 			// Already running subscription
 			if bool(req.Stop) {
+				// TODO: once we have a unsubscribe functionality, need to publish message to active sessions subject
```

Comment:
If we do end up having unsibscribe then we'd be probably better off just querying NATS.